unit-registry/core/pom.xml

```diff
@@ -129,5 +129,10 @@
             <artifactId>jul.extension.rct</artifactId>
             <version>${dependency.jul.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.openbase.bco</groupId>
+            <artifactId>authentication.core</artifactId>
```

Comment:
is is dep realy needed? Because the registry lib already includes the authentication.lib and mybe the core is not needed here.

Reply:
This dep is needed because the UserCreationPlugin needs the AuthenticationController for the initial password